releaf.gemspec

```diff
@@ -38,5 +38,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'spring-commands-rspec'
   s.add_development_dependency 'roo'
 
-  s.required_ruby_version = '>= 2.2.0'
```

Comment:
We only leave this line for releaf-core. There is no need to repeat it over and over for each gem.